swift-mode.el

```diff
@@ -225,7 +225,10 @@
              (looking-at "[ \n\t]+[.?:]")
              ;; Operators placed on the second line in multi-line expression
              ;; Should respect here possible comments strict before the linebreak
-             (looking-at (concat "\\(\/\/.*\\)?\n[[:space:]]*" swift-smie--operators-regexp))
+             (and (looking-at
+                   (concat "\\(\/\/.*\\)?\n[[:space:]]*" swift-smie--operators-regexp))
+                  (not (looking-at
+                        (concat "\\(\/\/.*\\)?\n[[:space:]]*" "//"))))
```

Comment:
I think string concatenation here is unnecessary. Also I think `not` branch here can be probably included into positive lookup regex. What do you think?